databricks_cli/execution_context/cli.py

```diff
@@ -0,0 +1,251 @@
+                   "the command and return immediately.")
+@click.option('--language', required=False, default="python", 
+              type=click.Choice(['python', 'scala', 'sql'], case_sensitive=False),
+              help="The language for the context.")
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def execute_command_cli(api_client, output, cluster_id, context_id, command, language, wait): #  NOQA
+    """
+    Executes a command within an existing execution context.
+    """
+    client = ExecutionContextApi(api_client)
+    result = client.execute_command(cluster_id, context_id, command, language)
+    if not wait:
+        click.echo(pretty_format(result))
+        return
+
+    command_id = result["id"]
+    _wait_for_command(client, output, cluster_id, context_id, command_id)
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help="Executes a single command within a new execution context.")
+@click.option('--output', default=None, help=CommandOutputType.help,
+              type=CommandOutputType())
+@click.option('--cluster-id', required=True, type=ClusterIdClickType(),
+              help=ClusterIdClickType.help)
+@click.option('--command', required=True, type=CommandStringType(),
+              help=CommandStringType.help)
+@click.option('--language', required=False, default="python", 
+              type=click.Choice(['python', 'scala', 'sql'], case_sensitive=False),
+              help="The language for the context.")
+@click.option('--wait', required=False, default=False, type=click.BOOL,
+              help="If true wait for command completion, otherwise schedule"
+                   " the command and return immediately.")
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def execute_command_once_cli(api_client, output, cluster_id, command, language, wait): #  NOQA
+    """
+    Executes a single command within a new execution context.
+    The execution context is automatically created and cleaned up.
+    """
+    client = ExecutionContextApi(api_client)
+
+    with ExecutionContext(api_client, cluster_id) as context:
+        result = client.execute_command(cluster_id, context.id, command, language)
+        if not wait:
+            click.echo(pretty_format(result))
+            return
+
+        command_id = result["id"]
+        _wait_for_command(client, output, cluster_id, context.id, command_id)
+
+
+def _wait_for_command(client, output, cluster_id, context_id, command_id):
+    while True:
+        status = client.get_command_status(cluster_id, context_id, command_id)
+        if not OutputClickType.is_json(output):
+            click.echo("Status: " + status["status"])
+        if status["status"] in ["Cancelled", "Error", "Finished"]:
+            click.echo()
+            _print_command_result(output, status)
+            break
+        time.sleep(1)
+
+
+def _print_command_result(output, status):
```

Comment:
this code misses error handling. please take a look at terraform provider implementation of command result parsing.